tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -4,30 +4,31 @@
 from ocs_ci.ocs import constants
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs.resources import storage_cluster
-from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.utility.utils import ceph_health_check
 
 
 @ignore_leftovers
 @tier1
 @polarion_id('OCS-1191')
 class TestAddCapacity(ManageTest):
     """
+
     Automates adding variable capacity to the cluster while IOs running
     """
     def test_add_capacity(self):
         """
-        Test to add variable capacity to the OSD cluster while IOs running
-        """
-        self.ceph_cluster = CephCluster()
+       Test to add variable capacity to the OSD cluster while IOs running
+       """
+        assert ceph_health_check(), 'health check failed'
         osd_size = storage_cluster.get_osd_size()
         result = storage_cluster.add_capacity(osd_size)
         pod = OCP(
             kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace']
         )
         pod.wait_for_resource(
-            timeout=300,
+            timeout=1200,
```

Comment:
if the timeout has to be increased so much, I think it worth reporting a BZ as it looks like a regression

Reply:
When we test it locally, it finishes quickly (within 300 sec) but in Jenkins, 300sec was not enough.

Reply:
+1 on Elad's comment

Reply:
My suggestion is to check the logs to know what exactly is happening during those 300 seconds w.r.t new osd pods. IIUC you don't have the logs from RUN ID: 1585643096. Can you check other runs (even on 4.3) where this test has failed and logs are available?

Reply:
@shreekarSS revert this change back to 300 seconds for now, we can see in some more runs.

Reply:
Ok @vasukulkarni

Reply:
With my analysis, I figured out that, the timeout was not the real issue. Most failed cases were in AWS pf, due to the worker node resource, it was suppose to be m5.4xlarge but got m4.xlarge. No failures in vmware pf since production yamls are used.

Reply:
Few failures were due to post health checking. So according to @ebenahar  comments, I'll move pre health checking in setup so that add capacity should skip if health is not OK